src/Employer/Employer.Web/Mappings/ReviewFieldIndicatorMapper.cs

```diff
@@ -59,77 +60,104 @@ public static class ReviewFieldIndicatorMapper
+        public static IReadOnlyList<ReviewFieldIndicatorViewModel> GetApplicationProcessFieldIndicators => new List<ReviewFieldIndicatorViewModel>
         {
             new ReviewFieldIndicatorViewModel(FieldIdentifiers.ApplicationMethod, Anchors.ApplicationMethodHeading, "Application process requires edit"),
             new ReviewFieldIndicatorViewModel(FieldIdentifiers.ApplicationUrl, nameof(ApplicationProcessEditModel.ApplicationUrl), "Enter the web address candidates should use to apply for this vacancy requires edit"),
             new ReviewFieldIndicatorViewModel(FieldIdentifiers.ApplicationInstructions, nameof(ApplicationProcessEditModel.ApplicationInstructions), "Explain the application process requires edit")
         };
 
-        public static IEnumerable<ReviewFieldIndicatorViewModel> MapFromFieldIndicators(IEnumerable<ReviewFieldIndicatorViewModel> reviewFieldIndicatorsForPage, List<ManualQaFieldIndicator> reviewFieldIndicators)
+        public static IEnumerable<ReviewFieldIndicatorViewModel> MapFromFieldIndicators(IEnumerable<ReviewFieldIndicatorViewModel> reviewFieldIndicatorsForPage, List<ManualQaFieldIndicator> reviewFieldIndicators, RuleSetOutcome automatedResult)
         {
             var selectedFieldIdentifiers = reviewFieldIndicators
                 .Where(r => r.IsChangeRequested)
                 .Select(r => r.FieldIdentifier)
                 .ToList();
 
-            return reviewFieldIndicatorsForPage.Where(r => selectedFieldIdentifiers.Contains(r.ReviewFieldIdentifier));
+
+            var qaFailureOutcomeIndicators = automatedResult.RuleOutcomes
+                                                        ?.SelectMany(d => d.Details)
+                                                        .Where(x => x.Score > 0) ?? Enumerable.Empty<RuleOutcome>();
+
+            // Add manual qa indicators
+            var indicators = reviewFieldIndicatorsForPage.Where(r => selectedFieldIdentifiers.Contains(r.ReviewFieldIdentifier)).ToDictionary(x => x.ReviewFieldIdentifier, y => y);
+
+            // Add automated qa indicators
+            foreach(var outcome in qaFailureOutcomeIndicators)
+            {
+                if (indicators.ContainsKey(outcome.Target))
+                {
+                    indicators[outcome.Target].Texts.Add(outcome.Narrative);
+                }
+                else
+                {
+                    var matchingField = reviewFieldIndicatorsForPage.SingleOrDefault(x => x.ReviewFieldIdentifier == outcome.Target);
+
+                    if (matchingField != null) // The field might not be on the current page.
+                    {
+                        matchingField.Texts[0] = outcome.Narrative; // Override default text as manual message not needed.
+                        indicators.Add(outcome.Target, reviewFieldIndicatorsForPage.Single(x => x.ReviewFieldIdentifier == outcome.Target));
```

Comment:
Could this potentially intermingle Auto QA messages for fields that are not on the screen with those that are.  Wonder if we should group these types of messages together and then just append them to the bottom on the indicators list.

Also would these render as a hyperlink. Think they should be standard text as there is field to link to.